### ISSUE TITLE
Insufficient information in ansible parsing errors

### ISSUE THREAD

**Issue by Petrox** [ansible/ansible#13435]
ansible 1.9.4
ubuntu 14.04

writing "creates:" in a shell task in a role somewhere...

```
$ time ansible-playbook -vvv something.yml 
ERROR: creates is not a legal parameter in an Ansible task or handler
```

Of course I have the editor open, but if someone else would see this (like someone commits a change with the error and the other user runs into the problem), they won't know where to look without grepping the whole environment...

Suggestion: include filename linenumber and task name in the error.

```
ERROR: roles/randomrolename/tasks/sometask.yml line 112: creates is not a legal parameter in an Ansible task or handler: "shell"
```

 Cheers!

**Comment by bcoca** [comment]
This is already the case in 2.0

**Comment by abadger** [comment]
We're not likely to backport that from 2.0 so I'm going to close this as fixed in 2.0.
